src/main/java/cz/it4i/parallel/SimpleOstravaParadigm.java

```diff
@@ -89,6 +88,11 @@ public void run() {
 	}
 
 	// -- Overriden method --
```

Comment:
Although this comment has not been added within the scope of this merge request, I have just spotted that it is no longer needed, please remove it. Thank you.